gensim/sklearn_integration/sklearn_wrapper_gensim_lsimodel.py

```diff
@@ -58,7 +58,7 @@ def set_params(self, **parameters):
     def fit(self, X, y=None):
         """
         For fitting corpus into the class object.
-        Calls gensim.model.LsiModel:
+        Calls gensim.models.LsiModel:
         >>>gensim.models.LsiModel(corpus=corpus, num_topics=num_topics, id2word=id2word, chunksize=chunksize, decay=decay, onepass=onepass, power_iters=power_iters, extra_samples=extra_samples)
```

Comment:
These params (`id2word`, `chunksize` etc) are associated with the LSI model. This change is in the file `sklearn_wrapper_gensim_lsimodel.py`. Since this change was so small (literally one word in a docstring), I added this change in this PR (PR concerning RP model wrapper) itself. 
There is also a similar change for LDA model [here](https://github.com/RaRe-Technologies/gensim/pull/1395/files#diff-596882bb3556903c76e8fb6944b62226R79). Should I remove these changes from this PR?